distributed/utils_test.py

```diff
@@ -423,8 +423,7 @@ def disconnect_all(addresses, timeout=3):
                 not pytest.config.getoption("--runslow"),
                 reason="need --runslow option to run")
 except (AttributeError, ValueError):
-    def slow(*args):
-        pass
+    slow = pytest.mark.slow
```

Comment:
How about this gets moved to somewhere in `tests`? Right now this is broken when `conftest.py` is missing (like in the PyPI tarballs.)